language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -225,15 +273,15 @@ genDefDataType curPkgId conName curModName tpls def =
         makeSer serDesc =
             ["export const " <> conName <> serHeader <> " ({"] ++
             map ("  " <>) (onHead ("decoder: " <>) serDesc) ++
-            ["});"]
+            ["})"]
```

Comment:
Why?

Reply:
Probably a mistake. Let me check.

Reply:
No, not a mistake. We can't assume the result of `makeSer` is going to be in a context where a trailing `;` is right (the variant case has a usage where it's not - the alternative of course is do surgery on it with something like `replace`).
```
  Corge: ({
    decoder: () => jtv.object({
      x: daml.Int.decoder(),
      y: daml.Text.decoder(),
    }),
  });,
```

Reply:
That an explanation makes sense. Happy to change it then.